docsteady/spec.py

```diff
@@ -168,6 +168,32 @@ def process_steps(self, test_script):
         return teststeps_sorted
 
 
+def get_lvv_details(key):
+    """
+    get LVV information from Jira
+    :param key: LVV jira Key
+    :return: needed parameter (first stage, only high level req, if available in Jira
+    """
+    lvv = dict()
+    lvv['high_level_req'] = []
+    if key != "":
+        resp = requests.get(
+            Config.ISSUE_URL.format(issue=key),
+            auth=Config.AUTH
+        )
+        if resp.status_code != 200:
+            print("Unable to download")
```

Comment:
Better to merge these two lines so they're easier to find in the output:
```suggestion
            print(f"Unable to download: {resp.text}")
```